documentation/MeetingMinutes/2024/MOM-2024-02-14.md

```diff
@@ -0,0 +1,134 @@
+    - **Discussion on Issue #100 "Claims to be used in oAuth Access tokens".**
+    - **Discussion on Issue #108 "Question to documentation update for security schemes".**
+    - **Discussion on Issue #104 "Alignment of Client Credential authentication / authorization using API Gateway Pattern".**
+
+3. Action Points
+4. AoB
+
+<br>
+
+## Meeting Technicallities
+
+Jesús offered that ownership of the meeting invite could be transferred to the new chair. Axel said that for now this can stay as it is.
+
+
+## Discussion on Consensus in Camara Working Groups
+
+Recapitulation of the previous discussion on this topic: Jesús and Diego said there was consensus achieved during a WG call that we continue working on https://github.com/camaraproject/IdentityAndConsentManagement/pull/113
+Herbert said that lazy-consensus is the Camara way. Shilpa said no consensus was achieved. 
+Axel proposed that consensus should be achieved on Github because not everybody can attend meetings. Diego said that is not the Camara way.
+Shilpa proposed that if there is an important decision suggested to be made at the next meeting, then this should be communicated a few days before the meeting. So, that people know about the importance of the next meeting and attend when otherwise they would not attend.
+We agreed that is must be clear whether consensus is achieved or not, and that we need to get better on this.
+
+## Discussion the Way Forward regarding the Profile
+
+Jesús recapitulated the last meeting where PR #113 was discussed and that consensus was achieved.
+Axel and Shilpa did not agree with consensus beeing achieved - see previous section.
+Axel said that he created the issue https://github.com/camaraproject/IdentityAndConsentManagement/issues/122 and kindly asked WG members to comment on the issue.
+Axel also created PR https://github.com/camaraproject/IdentityAndConsentManagement/pull/121 and asked people to review it.
+
+After some discussion we agreed that WG members comment on the issue and review the PR seeking resolution and a decision whether to use the DT proposal or the TEF proposal as our WG's policy document.
+Elizabeth suggested that if discussions on GitHub get too long, we would call for a side meeting, anounce it on the mailing this and seek consensus.
+
+Axel agreed with this proposal and no objections where heard.
+
+Axel asked Bjorn Hjelm for his/OIDF's opinion. Bjorn said that OIDF is here to help.
+
+We agreed that if neither the commenting on the issue nor the review of the PR lead to consensus, the WG seek a vote at TSC.
+
+
+## Discussion about offline-access
+
+Jesús said that this PR "just" moves documentation from some place (GSMA) to another and that this WG agreed on moving documentation to Camara as the "place of truth" for both GSMA and Camara.
+And example was given that some Commonalities document was moved to ICM and this is similar. Shilpa objected that moving a document from Camara to Camara is different.
```

Comment:
```suggestion
And example was given that some Commonalities document was moved to ICM and this is similar. Shilpa objected that moving a document from Camara to Camara is different.
Jesus disagrees because the AuthN-Auth Z document also contained information, for example, about authorization flows that were not agreed upon in the I&CM working group and other features that were not even discussed. And in fact, that document was reworked after being moved and even removed for the v0.2.0 release. Same process can be followed now.
```

Reply:
Let's leave it at that, but it looks like you're suggesting that the GSMA Opengateway technical stream is not an open process. I don't think the GSMA would agree. And, AFAIK, many of the companies represented in this WG were also in the GSMA technical stream. But I agree, we are both right.